include/openthread/coap.h

```diff
@@ -755,6 +804,9 @@ otError otCoapOptionIteratorGetOptionUintValue(otCoapOptionIterator *aIterator,
  */
 otError otCoapOptionIteratorGetOptionValue(otCoapOptionIterator *aIterator, void *aValue);
 
+// TODO
```

Comment:
`// TODO`?

Reply:
Was a reminder to do the documentation for the method... whose utilization has been implemented in the mean time XD.